Gaijinizer patch/Map041.txt

```diff
@@ -79,260 +81,268 @@ Remove mind corruption if(v[42] \>= 100) #Dargoth
+No way...
+
+So in the end there did you intentionally retain
+your power so you could save me?#Ron
 > END STRING
 
 > BEGIN STRING
 あの時も話しましたが…
 
 この身体は、過度の魔力放出には耐えられないのです。
 > CONTEXT: Map041/events/5/pages/1/68/Dialogue/yuuma
-I also talked about that time ...
-This body can not stand excessive magic release.#MTLed
+I mentioned this before but...
+
+This body cannot withstand an excessive discharge
+of magical power.#Ron
 > END STRING
 
 > BEGIN STRING
 彼女をビビらせて、なおかつ彼女が去るまで……
 
 この身体は維持しておかねばなりませんでした。
 > CONTEXT: Map041/events/5/pages/1/72/Dialogue/yuuma
-Let her beat and until she leaves ...
-This body had to be maintained.#MTLed
+I had to maintain that amount of energy so I could
+frighten and drive off Hectate.#Ron
 > END STRING
 
 > BEGIN STRING
 ですが、あの直前の通路では少し本気を出しましたよ？
 
 あわよくば貴方が諦めて、私のものになればと…
 > CONTEXT: Map041/events/5/pages/1/77/Dialogue/yuuma
-But, did you get a little serious in the aisle
-just before that? If you do well, you will give
-up and it will be mine ...#MTLed
+But I did put in a bit of effort when you faced me
+in that hallway just before. With a bit of luck, I
+would have been able to make you mine.#Ron
 > END STRING
 
 > BEGIN STRING
 ああ……♡あの時の貴方の抱き心地…♡
 
 もしよろしかったら、また私に抱かせてくださいませ♡
 > CONTEXT: Map041/events/5/pages/1/81/Dialogue/yuuma
-Oh ...... your Hug comfort of that time...♡
-If it's wrong, please let me have it again#MTLed
+Aaah... ♡ The feeling of holding you... ♡
+If you'd like, please let me hold you again. ♡#Ron
 > END STRING
 
 > BEGIN STRING
 ねぇ、アイア…\\|\\^
 > CONTEXT: Map041/events/5/pages/1/90/Dialogue/kazuya
-Hey, Ia ...\\|\\^#MTLed
+Hey, Lily...\\|\\^#Ron
 > END STRING
 
 > BEGIN STRING
 ………じゃなかった！ルシィ！
 > CONTEXT: Map041/events/5/pages/1/94/Dialogue/kazuya
-………Was not! Lucy!#MTLed
+... Sorry! I mean Lucy!#Ron
 > END STRING
 
 > BEGIN STRING
 ………る、ルシィってさ、
 
 その姿になる前は、どんな悪魔だったの？
 > CONTEXT: Map041/events/5/pages/1/96/Dialogue/kazuya
-... ...... Rusy, What kind of demon was it before
-it became that figure?#MTLed
+L- Lucy... What sort of demon were you before you
+had this form?#Ron
 > END STRING
 
 > BEGIN STRING
 なんだか聞いた感じだと
 
 ずいぶんと嫌ってるみたいだけど……
 > CONTEXT: Map041/events/5/pages/1/100/Dialogue/kazuya
-I felt like I heard something. 
-I seem to dislike it a lot.#MTLed
+It kinda feels like you really don't want me
+asking about that.#Ron
 > END STRING
 
 > BEGIN STRING
 あ、いや！！話したくないんだったらいいんだ！
 
 プライベートなことだし……！
 > CONTEXT: Map041/events/5/pages/1/106/Dialogue/kazuya
-Oh no!!I wish I did not want to talk!
-Spongebob It is......!#MTLed
+No, I mean...!! You don't have to talk about it if
+you don't want to! It's private info and all that.#MTLed
 > END STRING
 
 > BEGIN STRING
 構いませんよ。隠すようなことでもありませんし。
 
 どのみち、いずれ貴方には話そうと思っておりました。
 > CONTEXT: Map041/events/5/pages/1/110/Dialogue/yuuma
-I do not mind. It's not like hiding.
-I was thinking of talking to you anyway.#MTLed
+I do not mind. It's not as though I mean to keep
+it hidden. I had thought about telling you at some
+point anyway.#Ron
```

Comment:
A few minor corrections and adding a contraction:
>_**I don't mind. It's not as though I meant to keep
it hidden. I had thought about telling you at some
point anyways.**_